conanfile.txt

```diff
@@ -0,0 +1,10 @@
+[requires]
+nlohmann_json/3.11.2
+gegles-spdlog_setup/1.1.0
+cpptoml/0.1.1
+spdlog/1.13.0
+fmt/10.2.1
+
+[generators]
+CMakeDeps
+CMakeToolchain
```

Comment:
```suggestion
CMakeToolchain

[options]
spdlog/*:shared=True
```
spdlog does not really handle static linkage very well. Its logger registry is implemented as a global singleton, which gets erroneously re-initialized in each shared library and/or executable that has statically linked against it.